rero_ils/modules/holdings/api.py

```diff
@@ -302,34 +303,48 @@ def get_holdings_type_by_holding_pid(cls, holding_pid):
         return None
 
     @classmethod
-    def get_holdings_pid_by_document_pid(cls, document_pid):
+    def get_holdings_pid_by_document_pid(cls, document_pid, with_masked=True):
         """Returns holding pids attached for a given document pid."""
-        results = HoldingsSearch()\
+        es_query = HoldingsSearch()\
             .filter('term', document__pid=document_pid)\
-            .source(['pid']).scan()
-        for holding in results:
+            .source(['pid'])
+        if not with_masked:
+            es_query = es_query.filter(
+                'bool', must_not=[Q('term', _masked=True)])
+        for holding in es_query.scan():
             yield holding.pid
 
     @classmethod
-    def get_holdings_pid_by_document_pid_by_org(cls, document_pid, org_pid):
+    def get_holdings_pid_by_document_pid_by_org(cls, document_pid, org_pid,
+                                                with_masked=True):
         """Returns holding pids attached for a given document pid."""
-        results = HoldingsSearch()\
+        es_query = HoldingsSearch()\
             .filter('term', document__pid=document_pid)\
             .filter('term', organisation__pid=org_pid)\
-            .source(['pid']).scan()
-        for holding in results:
+            .source(['pid'])
+        if not with_masked:
+            es_query = es_query.filter(
+                'bool', must_not=[Q('term', _masked=True)])
+        for holding in es_query.scan():
             yield holding.pid
 
     @classmethod
-    def get_holdings_by_document_by_view_code(cls, document_pid, viewcode):
+    def get_holdings_by_document_by_view_code(cls, document_pid, viewcode,
+                                              with_masked=True):
         """Returns holding pids by document and view code."""
```

Comment:
```suggestion
        """Returns holding PIDs by document and view code."""
```

If it's not too much an hassle, change all pid in PID and pids in PIDs in the docstrings. It's easier to read and understand.